bin/awslocal

```diff
@@ -88,8 +88,14 @@ def prepare_environment():
     env_dict = os.environ.copy()
     env_dict['PYTHONWARNINGS'] = os.environ.get(
         'PYTHONWARNINGS', 'ignore:Unverified HTTPS request')
-    env_dict['AWS_DEFAULT_REGION'] = os.environ.get(
-        'AWS_DEFAULT_REGION', 'us-east-1')
+    if os.environ.get('DEFAULT_REGION'):
+        env_dict['AWS_DEFAULT_REGION'] = os.environ.get(
+            'DEFAULT_REGION')
+        msg = 'Environment variable "AWS_DEFAULT_REGION" will be overwritten by "DEFAULT_REGION"'
+        print('WARNING: %s' % msg)
```

Comment:
Can we make this an INFO message, and also only print it if `AWS_DEFAULT_REGION` is actually set in the environment?

Reply:
Sure